http-server/src/main/java/org/triplea/modules/error/reporting/ErrorReportController.java

```diff
@@ -27,15 +27,18 @@
   private final BiFunction<String, ErrorReportRequest, ErrorReportResponse> errorReportIngestion;
 
   public static ErrorReportController build(final AppConfig configuration, final Jdbi jdbi) {
+    final boolean isTest = configuration.getGithubApiToken().equals("test");
+
     final GithubIssueClient githubIssueClient =
         GithubIssueClient.builder()
             .uri(AppConfig.GITHUB_WEB_SERVICE_API_URL)
             .authToken(configuration.getGithubApiToken())
             .githubOrg(AppConfig.GITHUB_ORG)
             .githubRepo(configuration.getGithubRepo())
+            .isTest(isTest)
             .build();
 
-    if (githubIssueClient.isTest()) {
+    if (isTest) {
       Preconditions.checkState(!configuration.isProd());
```

Comment:
What about checking `!isTest || !configuration.isProd()` or `!(isTest && configuration.isProd())` instead of the if block?

Reply:
I was/am tempted, though I'm not sure it's actually simpler to reason about.

Reply:
Same, it's kind of a weird thing to check though